### PR TITLE
Bug/brainvision/nostim

### PR DESCRIPTION
**Pull request by bpinsard**
Small bugfix in case there is no events caught in the data.
Other part of software expects specific dimensions/shape for the events array.

I have a related question: which subset of events are extracted with regexp "^Mk\d+=(.*)" ?
Is the purpose is to only select stimulation or could there be other events of interest (eg. ttl from other device...) or they would fit better in another field of the raw data?
Or maybe their could be an event descritor added in this array?

Thanks

### CODE REVIEW

**Comment by teonlamont** [comment]
thanks for the fix.

the regexp is to capture any marker regardless of the type (e.g. Stimulus, Response, etc.)

**Comment by teonlamont** [comment]
I should clarify, we extract what's called the marker type. typically the events are listed as such:
```Mk2=Stimulus,S253,487,1,0``` where `S253` is the marker type. we strip the indicating letter off and store the numeric.

**Comment by dengemann** [comment]
@bpinsard which honor :)

File: mne/io/brainvision/brainvision.py [hunk 1]
```diff
@@ -340,7 +340,7 @@ def _read_vmrk_events(fname):
         except IndexError:
             pass
 
-    events = np.array(events)
+    events = np.array(events).reshape(-1,3)
```
Comment (dengemann) [inline mne/io/brainvision/brainvision.py]: leave one whitespace between 1 and 3 for pep8

**Comment by agramfort** [comment]
thanks @bpinsard !

**Pull request merged**